README.md

```diff
@@ -12,3 +12,53 @@ Grafana or the like.
 This is a [Haskell stack tools](https://docs.haskellstack.org/) based
 project.  Given a working `stack`, one should be able to execute
 `stack build` which will fetch all of the dependencies and build.
+
+After building, you will have several executables.
+
+teslauth: Authenticates to Tesla's API (with your credentials) and
+stores them in a local database.  _For obvious reasons_ treat this
+carefully.
+
+teslacatcher: Drains an MQTT topic to a sqlite database.
+
+tesladb: Runs continuiously polling Tesla's API to write to a database
+and MQTT, by default.
+
+tesladbfix: Filters out fields that have bad timestamps to a table
+named "rejects" in the database.
+
+## Setup and Run
+
+The first thing you'll need to do is auth.  This runs once initially,
+then needs to be run once a day with the -r flag to refresh.
```

Comment:
It's probably better to give the example of `-r` invocation as well.  It doesn't require any other arguments (other than path to the db) since it uses the existing auth and refresh token to refresh the auth.